scss/components/_control-bar.scss

```diff
@@ -81,6 +81,13 @@
 
     .oo-control-bar-item:focus .oo-icon {
       @include highlight-base();
+      @include highlight-tab();
+      opacity: 1 !important;
+    }
+
+    .oo-control-bar-item:hover .oo-icon {
+      @include highlight-base();
+      @include remove-highlight-tab();
```

Comment:
I think it would probably be better to also remove the glow on `focus` and keep just the box shadow. This way when you hover over a focused element you will see both the focused and hover styles.